media-gfx/iscan/iscan-3.62.0.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop flag-o-matic udev xdg-utils
+
+DESCRIPTION="EPSON Image Scan v3 for Linux"
+HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php https://gitlab.com/utsushi/utsushi"
+SRC_URI="https://support.epson.net/linux/src/scanner/imagescanv3/common/imagescan_${PV}.orig.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="graphicsmagick gui"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-libs/boost:=
+	media-gfx/sane-backends
+	media-libs/tiff
+	virtual/libusb:1
+	virtual/jpeg
+	gui? ( dev-cpp/gtkmm:2.4 )
+	graphicsmagick? ( media-gfx/graphicsmagick:=[cxx] )
+	!graphicsmagick? ( media-gfx/imagemagick:=[cxx] )
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/utsushi-0.$(ver_cut 2-3)"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.61.0-ijg-libjpeg.patch
+	"${FILESDIR}"/${PN}-3.61.0-imagemagick-7.patch
+)
+
+src_prepare() {
+	default
+
+	# Remove vendored libraries
+	rm -r upstream/boost || die
+	# Workaround for deprecation warnings:
+	# https://gitlab.com/utsushi/utsushi/issues/90
+	sed -e 's|-Werror||g' -i configure.ac || die
+	eautoreconf
+}
+
+src_configure() {
+	# Workaround for:
+	# /usr/lib64/utsushi/libutsushi.so.0: undefined symbol: libcnx_usb_LTX_factory
+	append-ldflags $(no-as-needed)
```

Comment:
could we fix the build system instead?

Reply:
There is nothing wrong in calling `eautoreconf`, especially if it allows respect `LDFLAGS` (by not overriding `-Wl,--as-needed`).

Reply:
It looks like links to libcnx-usb.so.0 and libcnx-hexdump.so.0 in libsane-utsushi.so.1 get "lost" during libtool install/relink process.

Reply:
How shall we proceed ?

Reply:
Resolving as no reaction.